io.catenax.battery.battery_pass/3.0.1/BatteryPass.ttl

```diff
@@ -0,0 +1,920 @@
+    bamm:characteristic :PowerCharacteristic.
+:originalPowerCapabilityLimits a bamm:Property;
+    bamm:name "originalPowerCapabilityLimits";
+    bamm:preferredName "original power capability limits"@en;
+    bamm:description "Performance ability of the high voltage battery according to limits i.e. how much power it can provide within given limits is describing a regulatory requirement."@en;
+    bamm:characteristic :PowerCharacteristic.
+:minVoltage a bamm:Property;
+    bamm:name "minVoltage";
+    bamm:preferredName "min voltage"@en;
+    bamm:description "Value of the minimal voltage the battery is rated for is describing a regulatory requirement."@en;
+    bamm:characteristic :VoltCharacteristic;
+    bamm:exampleValue "2"^^xsd:float.
+:nominalVoltage a bamm:Property;
+    bamm:name "nominalVoltage";
+    bamm:preferredName "nominal voltage"@en;
+    bamm:description "Value of the nominal voltage the battery is rated for is describing a regulatory requirement."@en;
+    bamm:characteristic :VoltCharacteristic;
+    bamm:exampleValue "4.3"^^xsd:float.
+:maxVoltage a bamm:Property;
+    bamm:name "maxVoltage";
+    bamm:preferredName "max voltage"@en;
+    bamm:description "Value of the maximum voltage the battery is rated for is describing a regulatory requirement."@en;
+    bamm:characteristic :VoltCharacteristic;
+    bamm:exampleValue "6"^^xsd:float.
+:cellInternalResistance a bamm:Property;
+    bamm:name "cellInternalResistance";
+    bamm:preferredName "cell internal resistance"@en;
+    bamm:description "The resistance offered by the cell in the flow of the current"@en;
+    bamm:characteristic :MilliohmCharacteristic;
+    bamm:exampleValue "45"^^xsd:decimal.
+:packInternalResistance a bamm:Property;
+    bamm:name "packInternalResistance";
+    bamm:preferredName "pack internal resistance"@en;
+    bamm:description "Total internal resistance in a battery pack is describing a regulatory requirement."@en;
+    bamm:characteristic :OhmCharacterisitic;
+    bamm:exampleValue "67"^^xsd:decimal.
+:packInternalResistanceIncrease a bamm:Property;
+    bamm:name "packInternalResistanceIncrease";
+    bamm:preferredName "pack internal resistanceIncrease"@en;
+    bamm:description "Increase in internal resistance of a battery pack over a period of time is describing a regulatory requirement."@en;
+    bamm:characteristic :PercentCharacteristic;
+    bamm:exampleValue "23"^^xsd:decimal.
+:maximumAllowedBatteryEnergy a bamm:Property;
+    bamm:name "maximumAllowedBatteryEnergy";
+    bamm:preferredName "maximum allowed battery energy"@en;
+    bamm:description "Maximum allowed battery energy (Wh) of the battery is describing a regulatory requirement."@en;
+    bamm:see <https%3A%2F%2Feur-lex.europa.eu%2Flegal-content%2FEN%2FTXT%2F%3Furi%3DCELEX%3A52020PC0798>;
+    bamm:characteristic :MaximumAllowedBatteryEnergyCharacteristic;
+    bamm:exampleValue "90000"^^xsd:float.
```

Comment:
same here, why float not decimal? also consider writing 9000.00 as example. Also other occurrences of datatype float and decimal

Reply:
done